modules/info.xq

```diff
@@ -0,0 +1,41 @@
+xquery version "3.1";
+
+try {
+    serialize(
```

Comment:
Have you considered replacing the 2 calls to `serialize()` with a single serialization declaration in the prolog?

Reply:
Yes, I about this. It is in all of the [XQuery modules](modules).  So why am I hesitant? Those modules should work _at least_ down to exist 4. 
Do you happen to know when exist started to support the following?
```
declare namespace output="http://www.w3.org/2010/xslt-xquery-serialization";

declare option output:method "json";
declare option output:media-type "application/json";
```

Reply:
I am going to test it right away. :)

Reply:
It does __not__ work! This might be due to the fact that this is executed through XML-RPC

Reply:
Wolfgang added the `output:method` option [in July 2012](https://markmail.org/message/ojasso6drkpbxemz), so it would've gone into eXist 2.0, released in [February 2013](https://sourceforge.net/projects/exist/files/Stable/2.0/). So you're safe to use it for the version range you're targeting and even earlier.

Reply:
If so, then yes, I'd bet it's a limitation (a bug perhaps?) in eXist's XML-RPC interface.

Reply:
XML-RPC needs to return XML so I am more inclined to call it a limitation of the protocol that is used (for now). Switching to the REST interface for these queries would be an option but that would also mean that an instance needs to have both channels open (XML-RPC and REST).

Reply:
I tried to use just
```
declare namespace output="http://www.w3.org/2010/xslt-xquery-serialization";
declare option output:method "json";
```
but this also does not work